### DISCUSSION TITLE
The text outside the screen border appears on the other side

### DISCUSSION THREAD

**Discussion by aivs** [olikraus/u8g2#1568] [Q&A]
Hi!
I use OLED display 256x64 with controller SSD1322 + STM32F108.
I'm trying to make a function to move text outside the screen border, but the text starts to appear on the other side.
How to avoid the appearance of text on the other side?
```
int main(void) {
    u8g2_Setup_ssd1322_nhd_256x64_f(&u8g2, U8G2_R0, u8x8_byte_4wire_hw_spi, u8x8_stm32_gpio_and_delay);
    u8g2_InitDisplay(&u8g2);
    u8g2_SetPowerSave(&u8g2, 0);
    while(1){
        u8g2_SetFont(u8g2, u8g2_font_profont29_tf);
        char str = "Value";
        for (int16_t x = 0; x<256; x++) {
            u8g2_DrawUTF8(u8g2, x, 0, str);
        }
    }
}
```

**Comment by olikraus** [comment]
You need to activate u8g2 16 bit mode, see faq.

**Comment by aivs** [comment]
Thanks, but it did't help. I've uncommented line in u8g2.h
```
#define U8G2_16BIT
```

The page goes to the right and appears completely on the left

**Reply by olikraus** [reply]
Uncommenting U8G2_16BIT *will* solve the problem. However you need to recompile everything. Additionally: Maybe you have modified the wrong library if you have multiple u8g2 lib versions on your hard disc.

**Comment by aivs** [comment]
Yes! With this code it's work!
I think the problem in my code.
```
      u8g2_ClearBuffer(&u8g2);
      u8g2_SetFont(&u8g2, u8g2_font_profont29_tf);
      u8g2_DrawUTF8(&u8g2, 0, 64, "Value");
      u8g2_SendBuffer(&u8g2);

      for (int16_t x = 0; x<256; x++) {
    	  u8g2_ClearBuffer(&u8g2);
          u8g2_DrawUTF8(&u8g2, x, 64, "Value");
          u8g2_SendBuffer(&u8g2);
      }
```
